src/main/java/com/solodev/clickergamebackend/controller/PlayerController.java

```diff
@@ -0,0 +1,29 @@
+package com.solodev.clickergamebackend.controller;
+
+import com.solodev.clickergamebackend.model.PlayerModel;
+import com.solodev.clickergamebackend.model.Token;
+import com.solodev.clickergamebackend.service.PlayerService;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@CrossOrigin(origins = "http://localhost:4200/")
```

Comment:
CORS errors only happen when FE and BE are both hosted on your own computer, which isn't the case here, so probably should remove it or comment it, because we don't want to allow it in production